Automattic/jetpack
# Package generator: Search and replace package name

#### Changes proposed in this Pull Request:

* Searches and automatically replace the project name in skeleton files

#### Other information:


#### Jetpack product discussion


NA
#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run `jetpack generate` to generate a new package
* Look at the new package created and confirm the `@package` annotation in `src/example.php` file has the right package name
* Also confirm that three values in `.phpcs.dir.xml` file have the right package name
